Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingAccelerationStructurePass.cpp

```diff
@@ -109,6 +112,16 @@ namespace AZ
                     }
                 }
 
+                // Attach output data from the skinning pass. This is needed to ensure that this pass is executed after
+                // the skinning pass has finished. We assume that the pipeline has a skinning pass with this output available.
+                if (rayTracingFeatureProcessor->GetSkinnedMeshCount() > 0)
```

Comment:
There is a perf implication here where we are now running RT passes after Skinning instead of before. I wonder if moving RT passes to async compute could be considered in future. Just a thought for now.